ci: run Renovate post tasks sequentially

alan-agius4 opened the pull request:
Due to a bug in Renovate (see: https://github.com/sarunint/renovate/blob/276a01fdd743d270fd2662cabb3b0e1e465aec83/lib/util/exec/common.ts#L50-L53), post tasks are incorrectly running in parallel. This causes 'yarn install' to overlap with 'yarn ng-dev misc update-generated-files', resulting in incomplete installs before file updates start.

josephperrott commented on package.json:
```diff
@@ -21,7 +21,7 @@
     "// 2": "Find the usage you are looking for with:",
     "// 3": "yarn ng-dev --help",
     "/ ": "",
-    "postinstall": "node --preserve-symlinks --preserve-symlinks-main ./tools/postinstall-patches.js && patch-package --patch-dir tools/esm-interop/patches/npm && patch-package --patch-dir tools/npm-patches",
```
Don't we still need to do the patch package work?

alan-agius4 replied:
Oops that’s a mistake

alan-agius4 replied:
Pushed

alan-agius4 committed:
ci: run Renovate post tasks sequentially

Due to a bug in Renovate (see: https://github.com/sarunint/renovate/blob/276a01fdd743d270fd2662cabb3b0e1e465aec83/lib/util/exec/common.ts#L50-L53), post tasks are incorrectly running in parallel. This causes 'yarn install' to overlap with 'yarn ng-dev misc update-generated-files', resulting in incomplete installs before file updates start.

josephperrott reviewed (approved):
LGTM

thePunderWoman commented:
This PR was merged into the repository by commit fbd7b7c48496476fa747acba61f62af98ceef3d0.

The changes were merged into the following branches: main, 19.0.x
